v2/protegrity-data-tokenization/src/main/java/com/google/cloud/teleport/v2/transforms/ProtegrityDataProtectors.java

```diff
@@ -298,12 +298,6 @@ private String formatJsonsToDsgBatch(Iterable<String> jsons) {
       CloseableHttpResponse response = sendToDsg(
           formatJsonsToDsgBatch(rowsToJsons(inputRows)).getBytes());
 
-      if (response.getStatusLine().getStatusCode() != HttpStatus.SC_OK) {
-        LOG.error("Send to DSG '{}' failed with '{}'",
-            this.dsgURI,
-            response.getStatusLine());
-      }
-
```

Comment:
Probably, you should try rebasing on master - this is a recent fix from Mikhail, we shouldn't delete it 🙂